test/support/utils.spec.js

```diff
@@ -0,0 +1,13 @@
+import { getNICEAccountsUrl } from "../../src/support/utils";
+
+describe("getNICEAccountsUrl", () => {
+
+	it("should return a URL with an appropriate domain", () => {
```

Comment:
This function should probably have validation (and tests) for valid environments - ie throw an error if not one of alpha, test, live etc - x5 or so?